packages/venia-concept/src/components/ProductOptions/option.js

```diff
@@ -30,11 +31,13 @@ class Option extends Component {
     };
 
     get listComponent() {
-        const { attribute_code } = this.props;
+        const { attribute_code, values } = this.props;
 
         // TODO: get an explicit field from the API
         // that identifies an attribute as a swatch
-        return attribute_code === 'fashion_color' ? SwatchList : TileList;
+        const optionType = getOptionType({ attribute_code, values });
```

Comment:
Do you need to pass `values` to this function? Didn't see it used in the implementation.

Reply:
It's not currently used in the implementation because the implementation is mocked. In the future, when `getOptionType` is capable of doing something, it will need both arguments.